docs/cloud/pouta/tutorials/image-pipeline.md

```diff
@@ -0,0 +1,650 @@
+| OS-SRV-USG:launched_at      | None                                                 |
+| OS-SRV-USG:terminated_at    | None                                                 |
+| accessIPv4                  |                                                      |
+| accessIPv6                  |                                                      |
+| addresses                   |                                                      |
+| adminPass                   | gCUbcRtu26Ka                                         |
+| config_drive                |                                                      |
+| created                     | 2025-02-04T13:10:43Z                                 |
+| flavor                      | standard.tiny (0143b0d1-4788-4d1f-aa04-4473e4a7c2a6) |
+| hostId                      |                                                      |
+| id                          | ae9f924b-f6c5-488c-b617-36809008e37e                 |
+| image                       | Ubuntu-24.04 (bc68d79a-6dcc-446f-a8cd-c8313b885718)  |
+| key_name                    | mykeypair                                            |
+| name                        | pipeline_vm                                          |
+| progress                    | 0                                                    |
+| project_id                  | ef20bcc4215d49ddbf33e5c5740d89fe                     |
+| properties                  |                                                      |
+| security_groups             | name='default'                                       |
+| status                      | BUILD                                                |
+| updated                     | 2025-02-04T13:10:43Z                                 |
+| user_id                     | timo                                                 |
+| volumes_attached            |                                                      |
++-----------------------------+------------------------------------------------------+
+```
+
+In the rest of the tutorial we assume that the name of the virtual machine in cPouta is `pipeline_vm`.
+
+## Step 4: configuring the pipeline
+
+Now that we have built all the components, we configure them to work as a pipeline.
+First, we configure the virtual machine to allow us to access it from our workstation.
+Then, we make sure that the virtual machine can work with the buckets in Allas, as well as that our database instance in Pukki accepts traffic coming from the virtual machine.
+Finally, we install and configure in the virtual machine the tools required to get the pipeline working.
+
+### Allowing traffic from workstation to virtual machine
+
+Access to the virtual machine is regulated by means of _security groups_ and the rules they contain.
+We thus create a new security group with a single rule, which allow access to the virtual machine, for example, from our workstation.
```

Comment:
Later the command in the example creates a rule that allows any IP `0.0.0.0/0`. The sentence above is misleading. 

I think that given that this is an advanced article, the user should be able to understand the procedure. In <https://docs.csc.fi/cloud/dbaas/cli/#creating-a-database-instance> it is explained.

Reply:
Also it should be explained the extra security that it gives. And the consequences of not doing so.

Reply:
Fix attempted in https://github.com/CSCfi/csc-user-guide/pull/2424/commits/ee48e095a5e4ee808c3ac1b956a5e88186057add and https://github.com/CSCfi/csc-user-guide/pull/2424/commits/740686c2d16a051b75d1508761153ebb413cd9b8